README.md

```diff
@@ -1,4 +1,14 @@
-# smarthome-localhomesdk
+# Local Home SDK Virtual Environment
+
+A mock Local Home Platform implemented as stubs for the Local Home SDK
+
+## Usage
+
+1. Place bundled Node Local Fulfillment app into `home-app` folder.
+2. `npm install`
+3. `npm run test`
+
+This is a mock Local Home Platform that depends on the Local Home SDK and a compiled [Local Fulfillment App](https://github.com/actions-on-google/smart-home-local)
 
```

Comment:
Since you're adding quite a bit of structure to the repo here, it'd be good to add a note for:

- references/issues
- contributions 
- license

See: https://github.com/actions-on-google/smart-home-nodejs/blob/master/README.md as an example. This will proactively help any folks who wander onto the repository.

Reply:
Done